libbeat/scripts/Makefile

```diff
@@ -57,13 +68,13 @@ endif
 ### BUILDING ###
 
 # Builds beat
-.PHONY: build
-build: $(GOFILES)
+.PHONY: ${BEATNAME}
+${BEATNAME}: $(GOFILES)
```

Comment:
The `.PHONY` line makes it always build it, even if there were no changes in `$(GOFILES)`. So in the current form it's kind of confusing, because when reading the Makefile the impression one gets is that the build is called only when the sources are changed.

That would be awesome (to do it only when the sources are changed), but it needs to include libbeat sources, which I think currently it doesn't. Should also include the vendor directory.